_component_design/tooltip.md

```diff
@@ -13,28 +13,304 @@ last-modified: 2017-08-17
+
+The container houses the border, pointer, and text/content. 
+
+The the tooltip will stand off from the back ground by: a lighter background color than the interface background, a slight rule (line) around the tooltip box, and a slight shadow around the tooltip so it seems “higher” or “raised up” from the items below it. 
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-composition.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Tooltip specs 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Spacing
+{: .hxSubSectionTitle}
+The tooltip component has varied widths based on the amount of content contained within its bounds but always relies on the following consistent spacing specifications:
+
+- 8px between vertical elements
+- 12px between not related elements 
+- max-width of 400px
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-1.svg){:width="100%"}
 
--   Activates when hovered on click.
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   The tooltip disappears when mouse hovers away, or when clicked in the icon.
-    Tooltip does not reactivate on click.
+### Text
+{: .hxSubSectionTitle}
 
--   Tooltip should only be used on icon images, text, and table rows; use when
-    see fit.
+- 14px Roboto Medium for headers
+- 14px Roboto Regular for body copy
 
--   Max width for the tooltip is 440px.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Text specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-2.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Tooltip"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Tooltips/tooltip.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Pointer and placement
+{: .hxSubSectionTitle}
+
+- The pointer is 16 px wide at the base.
+- The apex is 8 px from the side of the rectangle. 
+- The pointer should start 12 px from the corner.
+- The side from which the pointer protrudes depends on the location of the trigger on the screen.
+
+	- Always have the tooltip protrude toward the center of the screen
```

Comment:
I'm not entirely too sure 'protrude' is the right word here? Also put a full stop at the end of the sentence here

Reply:
Now looks like the following after output with edits. This includes edit from the previous comment as well as the next three.

Reply:
![image](https://user-images.githubusercontent.com/586682/31908386-4b34097a-b7fc-11e7-9c57-0e9c684ddf29.png)